Makefile

```diff
@@ -18,6 +18,8 @@ PARALLEL_E2E_TESTS          := 2
 GARDENER_REPO_ROOT          ?= $(REPO_ROOT)/../gardener
 SEED_NAME                   := provider-extensions
 SEED_KUBECONFIG             := $(GARDENER_REPO_ROOT)/example/provider-extensions/seed/kubeconfig
+# TODO: Remove this with bumping gardener/gardener to version > v1.102
```

Comment:
Can we link what is the issue/PR that fixes the OOM issues with golangci-lint@v1.60.2? 

We already vendoring v1.102. Let's write it as v1.103+ (see suggestion below) to make it more clear. 

```suggestion
# golangci-lint@v1.60.3 <ADD_REASONING_HERE>.
#
# TODO(dimitar-kostadinov): Drop this version overwrite when the registry-cache extension vendors github.com/gardener/gardener@v1.103+.
```